### PR TITLE
start testing new order functionality

### PR DESCRIPTION
**Pull request by bh2smith**
here is an example of how we can test our new order class. Note that it requires a some tweaking of the implementation.

cc @Richymaestro.

### CODE REVIEW

**Comment by Richymaestro** [comment]
Have to check if this works, having some problems with running the whole repo

**Review by Richymaestro — CHANGES_REQUESTED** [review]
Can I change some things?

**Comment by bh2smith** [comment]
> Can I change some things?

Yes absolutely. Are you not able to? I'll add you as a contributor.

File: README.md
Comment (bh2smith) [inline README.md:24]: yes good eye!

**Comment by Richymaestro** [comment]
Have to check if my code works, try to do that this week

File: src/test_order.py
Comment (bh2smith) [inline src/test_order.py]: Does this actually work without the brackets here?
```suggestion
        self.assertEqual(order.set_json(), expected_order)
```

File: src/order.py
Comment (bh2smith) [inline src/order.py]: So sad to see these back in here...

Comment (Richymaestro) [reply src/order.py]: Yes, my next point I think

File: .env.sample
Comment (bh2smith) [inline .env.sample]: I really don't think the quotes are necessary here.

File: src/order.py
Comment (bh2smith) [inline src/order.py]: Is there a reason for this?

File: src/test_order.py
Comment (bh2smith) [inline src/test_order.py:52]: Why load Json from something that is supposed to be json?

File: src/test_order.py
Comment (bh2smith) [inline src/test_order.py]: Is this used?

**Review by bh2smith — COMMENTED** [review]
Seems alright, but there are some very weird changes here.

File: src/test_order.py
Comment (Richymaestro) [reply src/test_order.py]: Only in the testing to get full results instead of partial with ... between beginning and end

File: src/order.py
Comment (Richymaestro) [reply src/order.py]: No, changed it!

File: src/test_order.py
Comment (Richymaestro) [reply src/test_order.py:52]: Outputs of both variables are the same when I use json.loads, otherwise punctuation is a bit different so tests fail

File: .env.sample
Comment (Richymaestro) [reply .env.sample]: check!

**Comment by Richymaestro** [comment]
Do you mind adding me on Discord so we can DM further?

**Comment by Richymaestro** [comment]
You have some time for more comments? Or could we close this PR :)

**Comment by bh2smith** [comment]
Yea looks good - we can merge this if you're happy with it, but it was originally my PR so it requires your approval.

**Review by Richymaestro — APPROVED** [review]

**Pull request merged**
